recipes/metaphlan/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
You should add a proper `setup.py` to your software so that it can be installed with `pip install`. Then this script becomes unnecessary.